apps/desktop/src/lib/baseBranch/baseBranchService.ts

```diff
@@ -29,7 +29,10 @@ export class BaseBranchService {
 	// Deduplicated push repo information.
 	readonly pushRepo = shallowDeduplicate(
 		derived(this.base, (base) => {
-			return base ? parseRemoteUrl(base.pushRemoteUrl) : undefined;
+			// TODO: We should not send `pushRemoteUrl` if it's the same as `remoteUrl`.
+			if (base?.pushRemoteUrl && base.pushRemoteUrl !== base.remoteUrl) {
+				return parseRemoteUrl(base.pushRemoteUrl);
```

Comment:
Is the comment still needed?
It seems like the condition already is taking care of what your TODO describes

Reply:
The comment is about the back end sending identical remoteUrl and pushRemoteUrl. I think we should only send pushRemoteUrl if it's different from remoteUrl, but I also think we should kick that can down the road.

Reply:
Let me clarify the comment!

Reply:
Gotcha 👍